diesel_bench/Cargo.toml

```diff
@@ -75,5 +75,5 @@ codegen-units = 1
 
 [patch.crates-io]
 quaint = { git = "https://github.com/prisma/prisma-engines", branch = "main" }
-diesel-async = { git = "https://github.com/weiznich/diesel_async", branch = "main" }
+diesel-async = { git = "https://github.com/wattsense/diesel_async", branch = "sync_wrapper_use_cache" }
```

Comment:
Can we change this back to the original repo + the master branch? I will take care to merge both PR's at the same time to not break the benchmarking.

Reply:
Sure, I just removed all Cargo.toml changes.